### DISCUSSION TITLE
Running in Proxmox in my Fanless Server in my RV

### DISCUSSION THREAD

**Discussion by sdacheson78** [SmoothWAN/SmoothWAN#51] [Show and tell]
Hello ALL

This project is awesome.  I usually just make a simple VM with lubuntu running speedify on my proxmox server sitting in front of my opnsense router.  

This changed the game for me.  I am running Starlink eth1, Tmobile eth2, VZ eth3.  I have proxmox take in the USB devices (eth2/3) and put them into bridges so my other VM's can access them directly as needed.  

I converted the VMware image in proxmox using the covert tool and it just worked.  
I'm running with 2G of memory(probably overkill), AES-NI on, 2 sockets and 2 cores.

THANK YOU THANK YOU THANKYOU.  I will be giving this to ALL my RV friends and work and live full time on the road.

![image](https://user-images.githubusercontent.com/63829494/182068140-d2aac2a8-d04b-43b5-a258-853014f6d609.png)

**Comment by labberdabber** [comment]
What server hardware are you using under Proxmox?

**Reply by sdacheson78** [reply]
sorry just saw this its a qotom 
Qotom 8 LAN Mini PC with 8th Gen Intel Celeron Core i3 i5 i7 Processor Onboard Advanced Firewall Router VM
with an i5 and 16G ram

**Reply by sdacheson78** [reply]
> sorry just saw this its a qotom Qotom 8 LAN Mini PC with 8th Gen Intel Celeron Core i3 i5 i7 Processor Onboard Advanced Firewall Router VM with an i5 and 16G ram

also I've just upgraded to a cluster with CEPH and I run the server above and 2 WYSE 5060's and that also works moving the VM around between all clustered PCs with each WAN and a LAN (12 VLANS) the same on each device and configured in proxmox.

**Reply by sdacheson78** [reply]
https://www.aliexpress.com/item/3256801909355201.html?spm=a2g0o.order_list.0.0.2a5718022bCuCN&gatewayAdapt=4itemAdapt

**Reply by labberdabber** [reply]
Thanks! Super helpful.

**Comment by RVCruiser** [comment]
'm so happy to have found this posting. I know it is a bit older, but I have been struggling with getting my QOTOM setup the best way possible. I too am a full time RV'er so I'm sure we share some of the same struggles. Do you have each of the connections on separate VLANS or just Static on the same network?  I wasn't sure if I should have them all with static IP's and only use one for DHCP for the entire network or not -- any help based on your own config would be  greatly appreciated.

**Reply by sdacheson78** [reply]
I have a bridge setup for each network I run in proxmox.

1 bridge for Verizon connection.  I then put the verizon NIC and add bridge to speedify
1 bridge for starlink connection.  Same as above
1 bridge for connection out to opnsense
1 bridge for each iot, clients, guest, services

Opnsense uses the connection to speedify as a WAN.  I then have 4 internal VLANS for services, iot, clients, guests that is run by speedify.

I like opnsense better, but use speedify for for its awesomeness.  With this I can also bypass speedify if I want and add more wans to opnsense when I want to go direct without vpn.

**Reply by RVCruiser** [reply]
Thank you so much. I'm working on this tonight to see how far I can get.

**Comment by TalalMash** [comment]
Added a guide here: https://smoothwan.com/proxmox/
